server/db/db-data.js

```diff
@@ -0,0 +1,124 @@
+const mathRandom = function getRandomInt(minParam, maxParam) {
+  const min = Math.ceil(minParam);
+  const max = Math.floor(maxParam);
+
+  return Math.floor(Math.random() * (max - min + 1) + min);
+};
+
+const authors = function pickAuthors() {
+  const names = ['Gillian C.', 'Chapman U.', 'Isaiah L.', 'Blair B.', 'Bentle P.', 'Menzie F.', 'Alexander R.', 'Walton S.', 'Rosemary D.', 'Wheeler I.', 'Britney P.', 'Morton M.', 'Damian E.', 'Duncan B.', 'Stan E.', 'Brooks A.', 'Ward O.', 'Benson T.', 'Ferris S.', 'Mitchell E.', 'Ulderico O.', 'Toscani Y.', 'Fiorenza B.', 'Lombardi V.', 'Amando X.'];
+  const index = mathRandom(0, 24);
+
+  return names[index];
+};
+
+const questionDate = () => {
+  let month = mathRandom(1, 12);
+  if (month < 10) {
+    month = `0${month.toString()}`;
+  }
+  let day = mathRandom(1, 27);
+  if (day < 10) {
+    day = `0${day.toString()}`;
+  }
+  const year = mathRandom(2018, 2019).toString();
+
+  return `${year}-${month}-${day}`;
+};
+
+const answerDate = (qDate) => {
+
+  let day = Number(qDate.slice(8, 10));
+  let month = Number(qDate.slice(5, 7));
+  let year = Number(qDate.slice(0, 4));
+
+  if (day < 24) {
+    day += 3;
+  } else {
+    day = 1;
+    if (month > 11) {
+      month = 1;
+      year += 1;
+    } else {
+      month += 1;
+    }
+  }
+
+  if (day < 10) {
+    day = '0' + day.toString();
+  }
+  if (month < 10) {
+    month = `0${month.toString()}`;
+  }
+
+  return `${year}-${month}-${day}`;
+};
+
+
+const string = 'Live-edge fam knausgaard butcher Helvetica three wolf moon beard air plant activated charcoal hoodie stumptown Food truck art party cold-pressed activated charcoal jianbing etsy drinking vinegar blog waistcoat man braid succulents Taxidermy sriracha kitsch gochujang mixtape photo booth mustache small batch shaman skateboard Photo booth readymade Roof umami swag before they sold out woke Vegan glossier tacos biodiesel hexagon hot chicken deep fried everyday carry four loko umami butcher beard man braid k.';
+
+const sellers = function generateSellers(sellersParam) {
+  const index = mathRandom(0, 71);
+  const listOfSellers = sellersParam.split(' ');
+
+  return listOfSellers[index] + listOfSellers[index + 1];
+};
+
+const text = function createQuestionOrAnswer(textParam) {
+  const start = mathRandom(0, 300);
+  const end = mathRandom(500, 350);
```

Comment:
Unsure how this one works,

Reply:
Math.random() *(350 - 500 + 1) +500) = Math.random() * (-149) + 500,

Reply:
Math.random() * (500 - 350 + 1) + 350 = Math.random() * (151) + 350, looks to be the same nvm.

Reply:
Thank you!